server/chat-plugins/helptickets.ts

```diff
@@ -1365,3 +1370,13 @@ export const commands: ChatCommands = {
 		`/helpticket delete [user] - Deletes a user's ticket. Requires: &`,
 	],
 };
+
+export const punishmentfilter: PunishmentFilter = (user, punishment) => {
```

Comment:
```suggestion
export const punishmentfilter: Chat.PunishmentFilter = (user, punishment) => {
```

Can we do this instead of exporting a new global type?

Reply:
Sure; although I think we should do that for loginfilters and statusfilters too.